doc/man7/openssl-core_numbers.h.pod

```diff
@@ -0,0 +1,56 @@
+=pod
+
+=head1 NAME
+
+openssl/core_numbers.h
+- OpenSSL provider interface numbers and function types
```

Comment:
for numbers maybe?

Reply:
Er, no?  But perhaps "dispatch numbers" is better than "interface numbers"

Reply:
identifiers and function wrappers?

Reply:
... especially since I use "dispatch number" everywhere else.  Done

Reply:
> identifiers and function wrappers?

Those are new terms that we have never used before in this context...  and "wrappers"?  that's doesn't connect with what they are, at least to me.

Reply:
Ok.. I cant think of anything better.